api/envoy/extensions/filters/http/dynamic_modules/v3/dynamic_modules.proto

```diff
@@ -22,6 +22,10 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 //
 // A module can be loaded by multiple HTTP filters, hence the program can be structured in a way that
 // the module is loaded only once and shared across multiple filters providing multiple functionalities.
+//
+// A dynamic module can opt into being a terminal filter with no upstream by setting ``terminal_filter`` to
```

Comment:
We are clarifying that multiple filters can exist in a module above, so let's do so here as well

```suggestion
// A dynamic module HTTP filter can opt into being a terminal filter with no upstream by setting ``terminal_filter`` to
```